lib/pengine/unpack.c

```diff
@@ -3981,9 +4014,9 @@ unpack_rsc_op(pe_resource_t *rsc, pe_node_t *node, xmlNode *xml_op,
         parent = uber_parent(rsc);
     }
 
-    pe_rsc_trace(rsc, "Unpacking task %s/%s (call_id=%d, status=%d, rc=%d) on %s (role=%s)",
-                 task_key, task, task_id, status, rc, pe__node_name(node),
-                 role2text(rsc->role));
```

Comment:
The history entries are sorted earliest first then processed one by one. Role here is redundant with the "... role on ..." log from the end of this function for the entry immediately before this, plus role doesn't make immediate sense when you're reading this message.

Reply:
`rsc->role` may change before the end of the function, so I wasn't sure it was redundant. However, if it doesn't make immediate sense or is unhelpful, yeah drop it.

Reply:
Right, that's what the "role on" log at the end of the function is for. So for each entry we see

    Unpacking ENTRY-ID ...
    RSC role on NODE after ENTRY-ID ...

and there's no need to repeat the previous entry's role result in the next entry's "Unpacking" log, especially since it will be the immediate next message.